.github/CODEOWNERS

```diff
@@ -477,26 +477,31 @@ x-pack/examples/files_example @elastic/kibana-app-services
 /x-pack/plugins/security_solution/common/detection_engine/schemas/alerts @elastic/security-detections-response-alerts
 /x-pack/plugins/security_solution/common/field_maps @elastic/security-detections-response-alerts
 
+/x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui @elastic/security-detections-response-alerts
 /x-pack/plugins/security_solution/public/detections/pages/alerts @elastic/security-detections-response-alerts
 
 /x-pack/plugins/security_solution/server/lib/detection_engine/migrations @elastic/security-detections-response-alerts
-/x-pack/plugins/security_solution/server/lib/detection_engine/notifications @elastic/security-detections-response-alerts
-/x-pack/plugins/security_solution/server/lib/detection_engine/schemas @elastic/security-detections-response-alerts
+/x-pack/plugins/security_solution/server/lib/detection_engine/rule_preview @elastic/security-detections-response-alerts
 /x-pack/plugins/security_solution/server/lib/detection_engine/rule_types @elastic/security-detections-response-alerts
 /x-pack/plugins/security_solution/server/lib/detection_engine/signals @elastic/security-detections-response-alerts
 /x-pack/plugins/security_solution/server/lib/detection_engine/routes/index @elastic/security-detections-response-alerts
 /x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals @elastic/security-detections-response-alerts
 
 ## Security Solution sub teams - Detections and Response Rules
+/x-pack/plugins/security_solution/common/detection_engine/fleet_integrations @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/prebuilt_rules @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/rule_management @elastic/security-detections-response-rules
 /x-pack/plugins/security_solution/common/detection_engine/rule_monitoring @elastic/security-detections-response-rules
-/x-pack/plugins/security_solution/common/detection_engine/schemas/common @elastic/security-detections-response-rules
-/x-pack/plugins/security_solution/common/detection_engine/schemas/request @elastic/security-detections-response-rules
-/x-pack/plugins/security_solution/common/detection_engine/schemas/response @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/rule_schema @elastic/security-detections-response-rules
```

Comment:
I'd like to add `@elastic/security-detections-response-alerts` here and on `/x-pack/plugins/security_solution/server/lib/detection_engine/rule_schema` since our area does a fair amount of work adding fields to the schema, migrating rules, etc. The rule schemas are one of the few places I think shared ownership makes sense so both teams get pinged.

Reply:
@marshallmain I thought you would like to stop owning it, but apparently, I misunderstood you. Shared ownership makes total sense to me here. Will add.